src/shared/pages/list/mpamBusinessUnits/listNames.tsx

```diff
@@ -0,0 +1,8 @@
+export const listNames = {
+    MPAM_BUS_UNITS_1UKVI: 'UKVI',
```

Comment:
This constant doesn't match the convention of the others, should it be sans UKVI?

Reply:
Yes I'll remove the UKVI